docs/run_test/platform.rst

```diff
@@ -9,7 +9,69 @@ Run tests on different platforms
 Run on Azure
 ------------
 
-Follow the same procedure in :doc:`Getting started with Azure <quick_run>`.
+VM's can be deployed on Azure using images from vhd, shared image
+gallery or marketplace. If multiple types are specified, the first
+non-empty type is picked in the following order :
+vhd, shared image gallery and marketplace.
+
+Running using vhd
+^^^^^^^^^^^^^^^^^
+To run using vhd, add the following to runbook :
+.. code:: yaml
```

Comment:
The only suggestion I would give is to add a blank line after a section heading (and before a code block); nevertheless, it compiles the same in rST. Except that, all look good to me!

Reply:
Awesome! Thanks for taking a look.